sdk/python/tests/integration/registration/test_inference.py

```diff
@@ -227,7 +228,8 @@ def test_view_with_missing_feature(features_df: pd.DataFrame) -> pd.DataFrame:
 def test_datasource_inference():
     # Create Feature Views
     date_request = RequestSource(
-        name="date_request", schema={"some_date": ValueType.UNIX_TIMESTAMP}
+        name="date_request",
+        schema=[Field(name="some_date", dtype=PrimitiveFeastType.UNIX_TIMESTAMP)],
```

Comment:
Same, keep a test using the dict style to make sure it keeps working.

Reply:
Done.